sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/GeneratedIdEntity.java

```diff
@@ -0,0 +1,12 @@
+package com.azure.cosmos;
```

Comment:
Please add Microsoft headers to this file.

Reply:
fixed